src/main/kotlin/nl/info/zac/app/zaak/ZaakKoppelenRestService.kt

```diff
@@ -118,12 +117,25 @@ class ZaakKoppelenRestService @Inject constructor(
         targetZaak: ZaakZoekObject,
         relationType: RelatieType
     ) =
-        (areBothOpen(sourceZaak, targetZaak) || areBothClosed(sourceZaak, targetZaak)) &&
+        checkForStatus(sourceZaak, targetZaak, relationType) &&
             sourceZaak.hasLinkRights() &&
             targetZaak.hasLinkRights() &&
             sourceZaak.isLinkableTo(targetZaak, relationType) &&
             targetZaak.hasMatchingZaaktypeWith(sourceZaak, relationType)
 
+    private fun checkForStatus(
```

Comment:
```suggestion
    private fun statusesAreAligned(
```
or
```suggestion
    private fun areConsistentlyOpenOrClosed(
```
or my favourite:
```suggestion
    private fun Zaak.haveSameStatusAs(targetZaak: ZaakZoekObject, relationType: RelatieType)
```

The last one allows for:
`sourceZaak.hasSameStatusAs(targetZaak, relationType)`